core/subsurface-qt/diveobjecthelper.h

```diff
@@ -15,6 +15,10 @@ class DiveObjectHelper {
 	Q_PROPERTY(int id MEMBER id CONSTANT)
 	Q_PROPERTY(int rating MEMBER rating CONSTANT)
 	Q_PROPERTY(int visibility MEMBER visibility CONSTANT)
+	Q_PROPERTY(int wavesize MEMBER wavesize CONSTANT)
```

Comment:
Now that grantlee is gone do we actually need that DiveObjectHelper thing?

Reply:
True, we don't a lot of the bells and whistles and if I built it from scratch I wouldn't go this way. But we still need to access all the stuff in the dive list via string names (as they come from the template), so I don't think we can get much leaner here.